analyzers/src/SonarAnalyzer.CSharp/Rules/ParametersCorrectOrder.cs

```diff
@@ -35,6 +35,16 @@ public sealed class ParametersCorrectOrder : ParametersCorrectOrderBase<Argument
     {
         private static readonly DiagnosticDescriptor rule =
             DiagnosticDescriptorBuilder.GetDescriptor(DiagnosticId, MessageFormat, RspecStrings.ResourceManager);
+
+        private static SyntaxToken? GetExpressionSyntaxIdentifier(ExpressionSyntax expressionSyntax) =>
+            expressionSyntax switch
+            {
+                IdentifierNameSyntax identifierNameSyntax => identifierNameSyntax.Identifier,
```

Comment:
Just to make it shorter, same for other cases
```suggestion
                IdentifierNameSyntax identifier => identifier.Identifier,
```

Reply:
Have made it shorter